samples/java-eventing-shopping-cart/src/main/proto/cart/shopping_cart_view_model.proto

```diff
@@ -23,6 +23,10 @@ import "cart/shopping_cart_domain.proto";
 import "google/api/annotations.proto";
 
 service ShoppingCartViewService {
+  option (akkaserverless.service) = {
+    type : SERVICE_TYPE_VIEW
+  };
```

Comment:
We don't need this for now. This is only needed when using the codegen and the eventing sample doesn't use it. Anyway, I will remove it for now since I will be pushing to this PR (soon).